vxquery-core/src/main/java/org/apache/vxquery/compiler/rewriter/RewriteRuleset.java

```diff
@@ -199,6 +200,8 @@
         normalization.add(new RemoveUnusedAssignAndAggregateRule());
         normalization.add(new PushValueIntoDatascanRule());
         normalization.add(new PushKeysOrMembersIntoDatascanRule());
+        normalization.add(new EliminateSubplanForSingleItemsRule());
+        //normalization.add(new PushIndexingValueIntoDatascanRule());
```

Comment:
Should this rule be uncommented? I don't see PushIndexingValueIntoDatascanRule being added to the ruleset.